bicep/network-new.bicep

```diff
@@ -327,13 +322,18 @@ resource ccwVirtualNetwork 'Microsoft.Network/virtualNetworks@2023-11-01' = {
   }
 }
 
+var peeringEnabled = contains(network,'vnetToPeer')
+var peeredVnetId = network.?vnetToPeer.?id ?? 'q/w/e/r/t/y/u/i/o'

```

Comment:
Try using an approved fake resource id for these instead.

see https://review.learn.microsoft.com/en-us/help/platform/reference-sensitive-identifier?branch=main